### PR TITLE
e2ee: add unit tests

### PR DESCRIPTION
**Pull request by fippo**
Both for just encode function as well as an end-to-end test.

### CODE REVIEW

**Comment by saghul** [comment]
❤️

**Comment by fippo** [comment]
```
   SyntaxError: /home/runner/work/lib-jitsi-meet/lib-jitsi-meet/modules/e2ee/Context.js: Identifier directly after number (60:27)
    58 |         // Must be incremented on every frame that is sent, can be reset on
    59 |         // key changes.
  > 60 |         this._sendCount = 0n;
```
That is an odd failure. This line isn't even in the change?

**Comment by fippo** [comment]
https://github.com/bcoin-org/bcrypto/issues/14#issuecomment-484199450 -- lets see...

File: modules/e2ee/Context.spec.js [hunk 1]
```diff
@@ -0,0 +1,174 @@
...
+    return s.trim();
+}
+*/
+
+/* TODO: more tests
+ * - delta frames
+ * - frame header is not encrypted
+ * - different sendCounts
+ * - different key length
+ * - ratcheting in decodeFunction
```
Comment (fippo) [inline modules/e2ee/Context.spec.js:24]: there is a test for this but there should be a test for "it errors when running out of window" :-)

**Review by saghul — APPROVED** [review]

**Pull request merged**